packages/core/__tests__/warnings.js

```diff
@@ -206,3 +206,27 @@ test('unterminated comments', () => {
     `"Your styles have an unterminated comment (\\"/*\\" without corresponding \\"*/\\")."`
   )
 })
+
+test('`css` opaque object passed in as `className` prop', () => {
+  // react-test-renderer doesn't stringify `className` (unlike react-dom)
+  // we need to pass dummy `css` prop so `jsx` tries to handle it (and stringify the `className`)
```

Comment:
Could we use react testing library so we're using react-dom here and encounter what will happen in the browser?

Reply:
used testing library